README.md

```diff
@@ -10,23 +10,7 @@ Platform details : [![Hardware](https://img.shields.io/badge/hardware-wiki-green
 
 ## Supported Host Platforms
 
-For more details on building the SDK on a host platform please check the **User Guide** specified below:.
-
-| Operating System | Documentation | GitHub main status | Downloads |
-| --------- | ----------- | ----------- | ----------- |
-| Windows | [Build Instructions](scripts/windows) | [![Build status](https://dev.azure.com/AnalogDevices/3DToF-rework/_apis/build/status/analogdevicesinc.ToF?branchName=main)](https://dev.azure.com/AnalogDevices/3DToF-rework/_build?view=runs&branchFilter=3310) | [![ToF evaluation kit installer](https://img.shields.io/badge/release-ToF_evaluation_kit_installer-blue.svg)](https://github.com/analogdevicesinc/ToF/releases/latest) |
-| Linux | [Build Instructions](doc/itof/linux_build_instructions.md) | [![Build status](https://dev.azure.com/AnalogDevices/3DToF-rework/_apis/build/status/analogdevicesinc.ToF?branchName=main)](https://dev.azure.com/AnalogDevices/3DToF-rework/_build?view=runs&branchFilter=3310) | [![ToF evaluation kit installer](https://img.shields.io/badge/release-ToF_evaluation_kit_installer-blue.svg)](https://github.com/analogdevicesinc/ToF/releases/latest) |
```

Comment:
This table makes sense to exists as the SDK can be built on different platforms. Showing which platforms (or operating systems) are supported is valuable and so are the status of the CI jobs for that platforms.

Reply:
Solved